src/babashka/cli.cljc

```diff
@@ -548,8 +548,20 @@
                   (map pad widths row))]
     (map pad-row rows)))
 
+(defn- expand-multiline-cells [rows]
+  (let [col-cnt (count (first rows))]
+    (->> (for [row rows]
```

Comment:
I kinda prefer not using `for` if you could easily replace it with `map` because performance of `for` is way slower due to the bulky code it generates

Reply:
Is performance really a concern for specific work?
It doesn't really matter; I'm guessing you stay away from `for` in general to avoid using it in specific cases where performance matters. 
I can adapt the code to what you prefer here.